webrender/src/platform/macos/font.rs

```diff
@@ -189,21 +209,53 @@ impl FontContext {
                     FontRenderMode::Mono => (false, false),
                 };
 
+                cg_context.set_allows_font_subpixel_positioning(true);
+                cg_context.set_should_subpixel_position_fonts(true);
+
                 cg_context.set_allows_font_smoothing(smooth);
                 cg_context.set_should_smooth_fonts(smooth);
                 cg_context.set_allows_antialiasing(antialias);
                 cg_context.set_should_antialias(antialias);
-                cg_context.set_rgb_fill_color(1.0, 1.0, 1.0, 1.0);
 
                 let rasterization_origin = CGPoint {
                     x: -metrics.rasterized_left as f64,
                     y: metrics.rasterized_descent as f64,
                 };
+
+                // CGFonts have to be drawn with black text on a white background
+                // After 10.11, there are two different glyphs depending on
+                // white on black or black on white. Gecko defaults to black on white.
+                cg_context.set_rgb_fill_color(1.0, 1.0, 1.0, 1.0);
+
+                let rect = CGRect {
+                    origin: CGPoint {
+                        x: 0.0,
+                        y: 0.0,
+                    },
+                    size: CGSize {
+                        width: metrics.rasterized_width as f64,
+                        height: metrics.rasterized_height as f64,
+                    }
+                };
+
+                // Fill white
+                cg_context.fill_rect(rect);
+
+                // Now draw black
+                cg_context.set_text_drawing_mode(CGTextDrawingMode::CGTextFill);
+                cg_context.set_rgb_fill_color(0.0, 0.0, 0.0, 1.0);
+
                 ct_font.draw_glyphs(&[glyph], &[rasterization_origin], cg_context.clone());
```

Comment:
Hmm, so previously the text was drawn with the white color, and now it's drawn with black. Is that intended?
We might need to run the servo test suite on Mac to see if it breaks.